### PR TITLE
Add brandID entry point and fix regex to match brand requirements

### PR DESCRIPTION
**Pull request by brumargues**

### CODE REVIEW

File: install.js [hunk 1]
```diff
@@ -40,12 +40,19 @@ var git = getGitFolderPath(root);
...
+var brandID = '';
 var regex = '(([a-z]+-[0-9]+:|merge).{6,})';
 var errorMsg = 'Your commit message must match the pattern <task_namespace>-<#task>: <text>’, or to be a merge message';
 
 if (packageJsonContent && packageJsonContent['commit-msg']) {
     var commitMsgObj = packageJsonContent['commit-msg'];
 
+    if (commitMsgObj['brand-id']) {
+        brandID = commitMsgObj['brand-id'];
+        regex = '((' + brandID + ')+-[0-9]{1,}:|(POC):|(M|m+erge))+[a-z-A-Z ]{10,}';
```
Comment (thedoctor46super) [inline install.js]: Would this allow the emojis?

File: install.js [hunk 2]
```diff
@@ -8,177 +8,206 @@ var exists = fs.existsSync || path.existsSync;
...
+  var regex = '(([a-z]+-[0-9]+:|merge).{6,})';
+  var errorMsg = 'Your commit message must match the pattern <task_namespace>-<#task>: <text>’, or to be a merge message';
+
+  if (packageJsonContent && packageJsonContent['commit-msg']) {
     var commitMsgObj = packageJsonContent['commit-msg'];
 
+    if (commitMsgObj['brand-id']) {
+      brandID = commitMsgObj['brand-id'];
+      branch_regex = '(feat|fix|docs|style|refactor|test|chore)\/(' + brandID + ')+-[0-9]{1,}-[a-z0-9]{1,}';
+      regex = '(((' + brandID + ')+-[0-9]{1,}|(POC\/[a-z0-9]{1,})): :+[a-z0-9]{1,}: |(M|m+erge))+[a-zA-Z0-9 ]{10,}';
```
Comment (thedoctor46super) [inline install.js]: Some of the emojis' names contain the `underscore` char, maybe you could add that to the regex

Comment (brumargues) [reply install.js]: Done

**Comment by stevencowles** [comment]
Should there be a package.json change for `commit-msg`?

File: install.js [hunk 3]
```diff
@@ -8,177 +8,206 @@ var exists = fs.existsSync || path.existsSync;
...
+      errorMsg = commitMsgObj['error-message'];
     }
-}
+  }
+
+  /**
+  * Bail out if we don't have an `.git` directory as the hooks will not get
+  * triggered. If we do have directory create a hooks folder if it doesn't exist.
+  */
+  if (!git) {
```
Comment (brumargues) [inline install.js]: Ah just realised that this existed, but was not working. Will remove it.

Comment (brumargues) [reply install.js]: Done.

**Pull request merged**